contracts/NodeDelegatorETH.sol

```diff
@@ -0,0 +1,378 @@
+                revert ValidatorAlreadyStaked(validators[i].pubkey);
+            }
+
+            _stakeEth(validators[i].pubkey, validators[i].signature, validators[i].depositDataRoot);
+            validatorsStaked[pubkeyHash] = true;
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    /// @dev Stake WETH and ETH in NDC in EigenLayer. It calls the `stake` function on the EigenPodManager
+    /// which calls `stake` on the EigenPod contract which calls `stake` on the Beacon DepositContract.
+    /// @dev The public functions that call this internal function are responsible for access control.
+    function _stakeEth(bytes calldata pubkey, bytes calldata signature, bytes32 depositDataRoot) internal {
+        // Call the stake function in the EigenPodManager
+        IEigenPodManager eigenPodManager = IEigenPodManager(lrtConfig.getContract(LRTConstants.EIGEN_POD_MANAGER));
+        eigenPodManager.stake{ value: 32 ether }(pubkey, signature, depositDataRoot);
+
+        // Increment the staked but not verified ETH
+        stakedButNotVerifiedEth += 32 ether;
+
+        emit ETHStaked(pubkey, 32 ether);
+    }
+
+    /// @notice Delegates all staked assets from this contract to an EigenLayer Operator.
+    /// This includes both both LSTs and native ETH.
+    /// @param operator the address of the EigenLayer Operator to delegate to.
+    function delegateTo(address operator) external onlyLRTManager {
+        revert("Unsupported");
+    }
+
+    /// @notice Undelegates all staked LST assets from this contract from the
+    /// previously delegated to EigenLayer Operator.
+    /// This also forces a withdrawal so the assets will need to be claimed.
+    function undelegate() external onlyLRTManager {
+        revert("Unsupported");
+    }
+
+    /// @dev Triggers stopped state. Contract must not be paused.
+    function pause() external onlyLRTManager {
+        _pause();
+    }
+
+    /// @dev Returns to normal state. Contract must be paused
+    function unpause() external onlyLRTAdmin {
+        _unpause();
+    }
+
+    /// @dev Approves the SSV Network contract to transfer SSV tokens for deposits
+    function approveSSV() external onlyLRTManager {
+        address SSV_TOKEN_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_TOKEN);
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        IERC20(SSV_TOKEN_ADDRESS).approve(SSV_NETWORK_ADDRESS, type(uint256).max);
+    }
+
+    /// @dev Deposits more SSV Tokens to the SSV Network contract which is used to pay the SSV Operators
+    function depositSSV(uint64[] memory operatorIds, uint256 amount, Cluster memory cluster) external onlyLRTManager {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).deposit(address(this), operatorIds, amount, cluster);
+    }
+
+    // @dev Withdraws SSV Tokens from a SSV Cluster
```

Comment:
Nit. a `/` is missing. This can break Etherscan function description (not 100% sure).

Reply:
Yes, this breaks Natspec. Fixed